app/javascript/controllers/patron_request_controller.js

```diff
@@ -1,82 +1,87 @@
 import { Controller } from "@hotwired/stimulus"
+import { Collapse } from "bootstrap"
 
 export default class extends Controller {
-  static targets = ['earliestAvailable', 'accordionButton']
+  static targets = ['earliestAvailable', 'accordion']
 
   connect() {
-    const form = this.element;
-    const accordionbuttons = this.accordionButtonTargets;
-
-    if (accordionbuttons.length == 1){
-      form.classList.remove('accordion');
-      const accordion = document.querySelector(`${accordionbuttons[0].dataset.bsTarget}-accordion`);
-      accordion.classList.remove('d-none');
-      accordion.classList.remove('shadow-sm');
-      accordion.querySelector('.accordion-header').remove();
-      const formitem = document.querySelector(`${accordionbuttons[0].dataset.bsTarget}`);
-      formitem.classList.remove('collapse')
+    if (this.accordionTargets.length == 1) {
+      this.removeAccordionStyling();
     } else {
-      accordionbuttons.forEach((accordionbutton, index) => {
-        accordionbutton.getElementsByClassName('step-number')[0].innerHTML = index + 1;
-        if (index == 0) {
-          accordionbutton.removeAttribute('disabled');
-          accordionbutton.click();
-        }
-      });
+      this.renumberVisibleSteps();
+      Collapse.getOrCreateInstance(this.accordionTargets[0].querySelector('.accordion-collapse')).show();
     }
   }
 
+  removeAccordionStyling() {
+    this.element.classList.remove('accordion');
+
+    const accordion = this.accordionTargets[0];
+    accordion.classList.remove('d-none');
+    accordion.classList.remove('shadow-sm');
+    accordion.querySelector('.accordion-header').remove();
+
+    Collapse.getOrCreateInstance(this.accordionTargets[0].querySelector('.accordion-collapse')).show();
```

Comment:
replacing with below seems to remove the accordion effect on the page.
const collpase = this.accordionTargets[0].querySelector('.accordion-collapse');
collpase.classList.remove('collapse');
collpase.classList.remove('accordion-collapse');

Reply:
I just pushed up [89b29a7](https://github.com/sul-dlss/sul-requests/pull/2198/commits/89b29a73dd426c142a96ad5643fdcedaf78ae703), which does all this at render-time.